gateway/gpio.go

```diff
@@ -0,0 +1,50 @@
+package gateway
```

Comment:
didn't think this code needed to be in a separate package anymore since only gateway uses it so moved here.